src/folderview.cpp

```diff
@@ -709,7 +709,7 @@ void FolderViewTreeView::layoutColumns() {
                     }
                 }
                 opt.section = columnId;
-                w = qMax(sizeHintForColumn(columnId),
+                w = std::max(sizeHintForColumn(columnId),
                          style()->sizeFromContents(QStyle::CT_HeaderSection, &opt, QSize(),
                                                    headerView).width());
```

Comment:
Please add four spaces before `style()->` and `headerView)` for them to be aligned with the changed line, like before.